src/module/ParticleCollector.cpp

```diff
@@ -69,7 +69,7 @@ void ParticleCollector::clearContainer() {
         container.clear();
 }
 
-std::vector<ref_ptr<Candidate> > ParticleCollector::getAll() const {
+std::vector<ref_ptr<Candidate> >& ParticleCollector::getContainer() const {
```

Comment:
I don't think so because in Python it works from the start as:
```python
for candidate in collector:
    ...
```
and that is recommended interface. Also, I don't think I ever publish examples with `.getAll()`. Only C++ users could be affected, but I don't think there are many of them. However, might be a good that we announce in the changelog that the `ParticleCollector` interface changed.

Reply:
I agree that probably nobody is affeccted.
@adundovi please just put a note in the changelog than I will approve the PR.

Reply:
Done. I also rebased this branch against the master branch.